<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>791</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>609</width>
      <height>789</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="mainLabel">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Convert2</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>25</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>by Gabriel Christo</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cheatsheetButton">
         <property name="text">
          <string>cheatsheet</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="xboxButton">
         <property name="text">
          <string>xbox supported formats</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>8</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Input File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputFile">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="selectInputButton">
           <property name="text">
            <string>select</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="infoButton">
           <property name="text">
            <string>file information</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Output File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="outputFile"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>7</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Video</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>codec</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="vCodecCombo">
              <item>
               <property name="text">
                <string>copy</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>libx264</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>libxvid</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>libvpx</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>preset</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="vPresetCombo">
              <item>
               <property name="text">
                <string>medium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ultrafast</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>superfast</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>veryfast</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>fast</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>slow</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>slower</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>veryslow</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>bitrate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="vBitrateCombo">
              <item>
               <property name="text">
                <string>copy</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>400k</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>800k</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1200k</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2M</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3M</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4M</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>resolution</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="resolutionCombo">
              <item>
               <property name="text">
                <string>copy</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>hd1080</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>hd720</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>hd480</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>vga</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>qvga</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Audio</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>codec</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="aCodecCombo">
              <item>
               <property name="text">
                <string>copy</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>aac</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ac3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>libvorbis</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>libopus</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>libmp3lame</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>bitrate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="aBitrateCombo">
              <item>
               <property name="text">
                <string>copy</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>192k</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>128k</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>64k</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>volume</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="aVolumeSpinbox">
              <property name="maximum">
               <number>150</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>%</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Downmix to Stereo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="downmixCheckbox">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Subtitle</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>insert</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="sCheckbox">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>srt file</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="sFilePath"/>
            </item>
            <item>
             <widget class="QPushButton" name="sSelectButton">
              <property name="text">
               <string>select</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>color</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="subColorCombo">
              <item>
               <property name="text">
                <string>white</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>yellow</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="subSizeSpinbox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="value">
               <number>25</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>px</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>additional parameters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="additionalCmd"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QPushButton" name="convertButton">
         <property name="text">
          <string>convert</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="crrtCmdButton">
         <property name="text">
          <string>show current command</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>8</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="output">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>350</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="documentTitle">
          <string notr="true"/>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QPushButton" name="clearOutputButton">
           <property name="text">
            <string>clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="killButton">
           <property name="text">
            <string>kill process</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
